src/cljs/main/broadfcui/common/icons.cljs

```diff
@@ -59,8 +60,14 @@
 
 (def fw-icon-width "1.28571429rem")
 
+(defn render-inline-icon [icon]
```

Comment:
Should this be a `defn-`? Not a leading question, just asking.

Reply:
I figured people might want to use it for one-off icons that aren't worth creating a function for